src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -101,8 +103,9 @@ public sealed partial class QuicStream : System.IO.Stream
         public override long Seek(long offset, System.IO.SeekOrigin origin) { throw null; }
         public override void SetLength(long value) { }
         public void Shutdown() { }
-        public System.Threading.Tasks.ValueTask ShutdownWriteCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public System.Threading.Tasks.ValueTask ShutdownCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask ShutdownWriteCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask StartCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
What's this for?

Reply:
For `QUIC_STREAM_EVENT_START_COMPLETE` notification. This will either survive or not depending how we decide to handle the stream opening, as described in https://github.com/dotnet/runtime/pull/52704#issuecomment-847089688